### PR TITLE
Javadoc cleanup 1203

### PR DESCRIPTION
**Pull request by ddunig2**
This PR is simply a backport of #6729 to the RxJava 2.x branch.

### CODE REVIEW

**Comment by akarnokd** [comment]
Whoa, that's definitively incorrect. Looks like you started with a very old commit. You should checkout 2.x from this repo, not your.older fork.

**Comment by akarnokd** [comment]
Okay, so it only needed retargeting. You have to select the target branch next time you create a PR for 2.x as the default is 3.x.

**Comment by ddunig2** [comment]
@akarnokd Yes my apologies, I did not click on the right branch. Do I have to make another PR to fix it or is it taken care of?

**Comment by akarnokd** [comment]
No need for a new PR, I could change the target to 2.x and it is now okay.

**Comment by ddunig2** [comment]
@akarnokd Can you rephrase that for me, please?

**Comment by akarnokd** [comment]
No need for a new PR, I could change the target to 2.x and it is now okay.

**Comment by ddunig2** [comment]
@akarnokd Okay great, nothing else left for me to do on this particular issue?

**Comment by akarnokd** [comment]
I don't think so, unless one of our regular reviewers finds something I overlooked.

**Comment by ddunig2** [comment]
@akarnokd ok great. What else can I help with?

**Comment by akarnokd** [comment]
Can't think of anything else right now.
